gym/core.py

```diff
@@ -70,7 +70,7 @@ def step(self, action: ActType) -> Tuple[ObsType, float, bool, dict]:
         raise NotImplementedError
 
     @abstractmethod
-    def reset(self, seed: Optional[int] = None) -> ObsType:
+    def reset(self, seed: Optional[int] = None , return_info: Optional[bool] = False) -> Union[ObsType,Tuple[ObsType, dict]]:
```

Comment:
It shouldn't be Optional[bool], just bool (but still with a default of False)

Despite the appearances, `Optional` doesn't mean that the argument is optional, but that it can also take the value `None`, which doesn't make sense here

Reply:
Good catch, I will fix this.